nemo/collections/asr/parts/submodules/rnnt_decoding.py

```diff
@@ -346,15 +348,35 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int, supported_punctu
 
         if strategy in {TransducerDecodingStrategyType.GREEDY, TransducerDecodingStrategyType.GREEDY_BATCH}:
             ngram_lm_model = self.cfg.greedy.get('ngram_lm_model', None)
+            ngram_lm_alpha = self.cfg.greedy.get('ngram_lm_alpha', 0)
+            boosting_tree = self.cfg.greedy.get('boosting_tree', None)
+            boosting_tree_alpha = self.cfg.greedy.get('boosting_tree_alpha', 0)
         else:
             ngram_lm_model = self.cfg.beam.get('ngram_lm_model', None)
+            ngram_lm_alpha = self.cfg.beam.get('ngram_lm_alpha', 0)
+            boosting_tree = self.cfg.beam.get('boosting_tree', None)
+            boosting_tree_alpha = self.cfg.beam.get('boosting_tree_alpha', 0)
+
+        # load fusion models from paths (ngram_lm_model and boosting_tree_model)
+        fusion_models, fusion_models_alpha = [], []
+        if ngram_lm_model is not None and ngram_lm_model.endswith('.nemo'):
```

Comment:
We also support `arpa` models
It looks like it is better to check the strategy here instead of checking for `.nemo` extension